usagov/usagov-2021
# USAGOV-1621-taxonomy-perms: Gives content admins permission to edit new benefits categories terms.

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1621

## Description

Grants the "administer taxonomy" permission to the content administrator role

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Confirm that a user with the content administrator role see the menus to go to the page for managing benefits categories terms. Drupal will show all vocabularies to them and restrict which terms they can create, update, and delete. Only the Benefits Categories vocab has links to create, edit, delete terms.

![image](https://github.com/usagov/usagov-2021/assets/493854/b921d64b-d689-4dd6-8629-adda227fc9b6)

### Requires New Config

### Requires New Content

### Validation Steps
- Log in as a user with the "Content Administrator" role and no other role.
- In Drupal admin, should see menu links for STructure > Taxonomy and menu links for all vocabularies
- Clink on link for Benefit Categories vocabulary, show show list of terms
- User should be able to add, edit, delete and re-order terms.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
